src/test/java/api/loans/scenarios/CheckInWithdrawnItemTest.java

```diff
@@ -0,0 +1,113 @@
+package api.loans.scenarios;
+
+import static api.support.matchers.ItemMatchers.available;
+import static api.support.matchers.ItemMatchers.awaitingPickup;
+import static api.support.matchers.ItemMatchers.inTransit;
+import static api.support.matchers.ItemMatchers.withdrawn;
+import static api.support.matchers.JsonObjectMatcher.hasJsonPath;
+import static api.support.matchers.LoanMatchers.hasItem;
+import static com.jayway.jsonpath.matchers.JsonPathMatchers.hasNoJsonPath;
+import static org.hamcrest.CoreMatchers.allOf;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.junit.Test;
+
+import api.support.APITests;
+import api.support.CheckInByBarcodeResponse;
+import api.support.builders.ItemBuilder;
+import api.support.builders.RequestBuilder;
+import api.support.http.InventoryItemResource;
+import io.vertx.core.json.JsonObject;
+
+public class CheckInWithdrawnItemTest extends APITests {
+  @Test
+  public void canCheckInAtHomeLocation() {
+    final InventoryItemResource item = itemsFixture
+      .basedUponSmallAngryPlanet(ItemBuilder::withdrawn);
+
+    assertThat(item.getJson(), withdrawn());
+
+    final CheckInByBarcodeResponse response = loansFixture.checkInByBarcode(item);
+
+    assertThat(response.getJson(), allOf(
+      hasNoJsonPath("loan"),
```

Comment:
What does this check mean, that there is no open loan once the item has been checked in?

Reply:
I've removed the check at all, because the main requirement is to be able to check-in such an item and assert that status changed.